_accounts.md

```diff
@@ -61,7 +61,7 @@ Retrieves a list of accounts for the current user.
 
 `GET https://api.uphold.com/v0/me/accounts`
 
-<aside class="notice">Requires the `accounts:read` scope for Uphold Connect applications.</aside>
+<aside class="notice">Requires the <code>accounts:read</code> scope for Uphold Connect applications.</aside>
```

Comment:
Some of these blocks have been expanded, while others remained inlined. What is the criteria?

Reply:
Line width, mostly, though I didn't validate that exhaustively. Happy to expand them all, for consistency. WDYT?

Reply:
Update: I've wrapped all of them; that way things remain consistent and we avoid potential back-and-forth (cf. changes done in #170's 8645259e660b3723c72f74f0e731acf2ceedad1f).